src/mds/MDSDaemon.cc

```diff
@@ -838,6 +838,9 @@ int MDSDaemon::_handle_command(
       cmd_getval(cct, cmdmap, "heapcmd", heapcmd);
       vector<string> heapcmd_vec;
       get_str_vec(heapcmd, heapcmd_vec);
+      string value;
+      if(cmd_getval(cct, cmdmap, "value", value))
```

Comment:
please update `doc/man/8/ceph.rst` accordingly. and might want to note what this option means, for instance,

> TCMalloc releases no-longer-used memory back to the kernel gradually. ``release_rate`` controls how quickly this happens. Increase this setting to make TCMalloc to return unused memory more frequently. 0 means never return memory to system, 1 means wait for 1000 pages after releasing a page to system. It is ``1.0`` by default.

and how is related to "release" command.